docs/user-guide/discovery-kernel-arguments-api.md

```diff
@@ -0,0 +1,82 @@
+# Discovery Kernel Arguments API
+
+The way to update discovery kernel arguments is by updating InfraEnv.  Currently, only **append** (additional argument) operation is supported.
```

Comment:
```suggestion
# Modifying Kernel Arguments for the Live ISO

Update the InfraEnv resource to modify the kernel arguments used in the live ISO (during the host discovery phase) .  Currently, only the **append** (additional argument) operation is supported.
```

I suggest also modifying the name of the file, people probably won't guess what "discovery kernel arguments" means.